src/app/components/dropdown.jsx

```diff
@@ -6,6 +6,7 @@ class DropDownComponent extends React.Component {
   componentWillMount () { this.setState({});}
 
   handleChange (selected) {
+    console.log("dropdown selected: " + selected);
```

Comment:
Let's not leave these in when merging to master, unless needed for ongoing debugging.  Clutters the console. @yshen01